tests/page-objects/common/common.po.js

```diff
@@ -24,7 +24,8 @@ const medicLogo = element(by.className('logo-full')),
   settings = element.all(by.css('.configuration a>span')),
   // Report bug
   bugDescriptionField = element(by.css('[placeholder="Bug description"]')),
-  modalFooter = element(by.css('.modal-footer'));
+  modalFooter = element(by.css('.modal-footer')),
+  deleteButton = $('[ng-show="!status.processing"]');
```

Comment:
This selector feels too generic to me, and brittle as we may decide to change the name of the `status` variable for no good reason. I think this would be much more reliable: `.modal-footer .submit`. Also note that all modals have the same structure so it might be a good idea to make sure you get the right one, in this case: `#delete-confirm .modal-footer .submit`.